src/services/conversation.py

```diff
@@ -13,9 +13,10 @@
     InitialQuestionsHandler,
 )
 from services.k8s import IK8sClient
+from utils.config import Config
 from utils.logging import get_logger
 from utils.models.factory import IModel, IModelFactory, ModelFactory, ModelType
-from utils.settings import REDIS_URL
+from utils.settings import REDIS_HOST, REDIS_PORT
```

Comment:
Check if we should use REDIS_URL? As this [PR](https://github.com/kyma-project/kyma-companion/pull/341) is merged now.

Reply:
I used REDIS_DB_NUMBER created. AsyncRedisSaver doesn't accept REDIS_URL.